src/components/Affidavit/Affidavit.vue

```diff
@@ -325,7 +332,7 @@ export default class Affidavit extends Mixins(CommonMixin, DocumentMixin, Entity
   private created (): void {
     this.uploadAffidavitDoc = this.getAffidavitStep.affidavitDoc as File
     this.uploadAffidavitDocKey = this.getAffidavitStep.docKey
-    this.affidavitConfirmed = !!this.getAffidavitStep.affidavitConfirmed
```

Comment:
Double exclamation mark is useful to cast a non-boolean to a boolean value. In this case, the right hand side value is already a boolean.